### DISCUSSION TITLE
automatic refresh of extensions-update in Joomla itself

### DISCUSSION THREAD

**Discussion by Rolli1962** [akeeba/panopticon#327] [General]
Have found out - if a new version of an extension is available those is only shown in panopticon, if there was an access via the back-end of the Joomla-site.

Means panopticon still collecting the status of each site, but those is outdated. Have compared it with the sites, where I did a login via back-end and where not. Even I wait a day and longer the update is not presented in panopticon.

Is it possible to activate something in Joomla to update the status of extensions-updates itself?
Maybe I have to set a parameter, where I am not aware of.

Even maybe those behaviour is special to J5.0.0 and those refresh was also done by a api-access in J4.x. For me even clear that those refresh couldn't be done 'hidden' if a visitor do access the website. Than it seems for him the site would be hanging for a while.

**Comment by nikosdion** [comment]
Yeah, Joomla! is a bit daft. It won't refresh the update information without being asked to – which is what happens when you visit the backend. I addressed that in https://github.com/akeeba/panopticon-connector/commit/80e1605ea9a23c1746a9bcd88b97b200a32096b5  A new release will be emitted next week.

**Comment by Rolli1962** [comment]
Thanks - is working with panopticon-connector 1.02 perfectly !

**Reply by nikosdion** [reply]
Awesome :) Thank you for the feedback and have a wonderful day!
